projects/weather-app.md

```diff
@@ -96,8 +97,8 @@ When you press the submit button on the form you will get data from the API. Mak
 
 ## Guide Code
 
-You can see a guide code working with hard-code data [here](https://github.com/Yosolita1978/HardCodeDataWeatherApi)
-You can see a guide code working with real data with a API_KEY [here](https://github.com/Yosolita1978/RealDataWeatherAPI)
+- You can see a guide code working with hard-code data [here](https://github.com/Yosolita1978/HardCodeDataWeatherApi).
```

Comment:
You can see an example of working with hard-coded data [here]

Reply:
Thanks, @mx-ruthie! Just committed that change and the commit # is d593b86152fb5b294a28c548606525bd937c7113